fixtures/kelvin/epos-300.json

```diff
@@ -0,0 +1,1107 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "Epos 300",
+  "categories": [
+    "Dimmer",
+    "Color Changer",
+    "Strobe"
+  ],
+  "meta": {
+    "authors": [
+      "Bjorn"
+    ],
+    "createDate": "2023-10-09",
+    "lastModifyDate": "2023-10-09"
+  },
+  "links": {
+    "manual": [
+      "https://www.kelvinlight.com/app/uploads/2023/08/Epos-300-User-Manual-EN-V1.0-May-2023-2.pdf",
+      "https://github.com/KelvinLights/Kelvin-DMX-Protocol/"
+    ],
+    "productPage": [
+      "https://www.kelvinlight.com/product/epos_300_full_color_spectrum_rgbacl_led_studio_light/"
+    ]
+  },
+  "physical": {
+    "dimensions": [
+      17,
+      33,
+      13
+    ],
+    "weight": 7,
+    "power": 300,
+    "DMXconnector": "5-pin",
+    "bulb": {
+      "type": "LED"
+    }
+  },
+  "availableChannels": {
+    "Main intensity": {
+      "name": "Main intensity",
+      "defaultValue": 0,
+      "fineChannelAliases": [
+        "Main intensity fine"
+      ],
+      "capability": {
+        "type": "Intensity"
+      }
+    },
+    "Output type 8bit": {
+      "name": "Output type",
+      "defaultValue": 0,
+      "capabilities": [
+        {
+          "type": "Generic",
+          "dmxRange": [
+            0,
+            0
+          ],
+          "comment": "Constant output"
+        },
+        {
+          "type": "Generic",
+          "dmxRange": [
+            1,
+            255
+          ],
+          "comment": "Maximum output"
+        }
+      ]
+    },
+    "CCT": {
+      "name": "CCT",
+      "defaultValue": 0,
+      "fineChannelAliases": [
+        "CCT fine"
+      ],
+      "capability": {
+        "type": "ColorTemperature",
+        "colorTemperatureStart": "1700K",
+        "colorTemperatureEnd": "20000K"
+      }
+    },
+    "Green-magenta point 8bit": {
+      "name": "Green-magenta point",
+      "defaultValue": 0,
+      "capabilities": [
+        {
+          "type": "Generic",
+          "dmxRange": [
+            0,
+            124
+          ],
+          "comment": "Neutral"
+        },
+        {
+          "type": "Generic",
+          "dmxRange": [
+            125,
+            255
+          ],
+          "comment": "-0.027 to 0.027"
+        }
+      ]
+    },
+    "Green-magenta point 16bit": {
+      "name": "Green-magenta point",
+      "fineChannelAliases": [
+        "Green-magenta point 16bit fine"
+      ],
+      "defaultValue": 0,
+      "capabilities": [
+        {
+          "type": "Generic",
+          "dmxRange": [
+            0,
+            31906
+          ],
```

Comment:
Please format arrays like other files in this repo, see also https://github.com/OpenLightingProject/open-fixture-library/blob/master/lib/fixture-json-stringify.js.

I already tried to do this myself, but it seems like I don't have the permission to push on your branch.

Reply:
will do, thanks a lot for the assistance in getting this through